LibOS/shim/test/regression/epoll_wait_timeout.c

```diff
@@ -0,0 +1,119 @@
+/* -*- mode:c; c-file-style:"k&r"; c-basic-offset: 4; tab-width:4; indent-tabs-mode:nil; mode:auto-fill; fill-column:78; -*- */
+/* vim: set ts=4 sw=4 et tw=78 fo=cqt wm=0: */
+
+#include <errno.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <sys/epoll.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netdb.h>
+
+static int create_and_bind (char *port) {
```

Comment:
Consider adding a comment about how to interpret the return value.  This function also seems to have some failure cases that do not print a clear error.